diff -up gdm-2.28.0/data/session-setup.entries.bubble-location gdm-2.28.0/data/session-setup.entries
--- gdm-2.28.0/data/session-setup.entries.bubble-location	2009-10-09 09:38:01.315620045 -0400
+++ gdm-2.28.0/data/session-setup.entries	2009-10-09 09:39:11.354874396 -0400
@@ -755,5 +755,11 @@
         <string>/bin/true</string>
       </value>
     </entry>
+    <entry>
+      <key>/apps/notification-daemon/popup_location</key>
+      <value>
+        <string>bottom_right</string>
+      </value>
+    </entry>
   </entrylist>
 </gconfentryfile>
